docker/compose/sawtooth-default.yaml

```diff
@@ -72,6 +76,6 @@ services:
     depends_on:
       - rest-api
     entrypoint: "bash -c \"\
-        sawtooth keygen && \
+        (test -e /etc/sawtooth/keys/validator.priv || sawtooth keygen) && \
```

Comment:
I think this should be testing for the user key at `/root/.sawtooth/keys/root.priv`

Reply:
@rbuysse is correct. `sawtooth keygen` generates `$HOME/.sawtooth/keys/$USER.priv` and `sawadm keygen` generates `/etc/sawtooth/keys/validator.priv`